Features/Standard/SubitemGrid.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Subitem Grid
+
+    @javascript
+    Scenario: The Subitem Grid view is used when viewing a Content item which Content Type belongs to the Media Content Type Group
+        Given a Content Type of the Media Content Type Group marked as container
```

Comment:
I think that our domain would still work phrased as "of the Media Group".

I'd maybe prefer "flagged" over "marked". Both are correct, but I like "flagged" for boolean options.